blocks/ai-chat/src/complete-chat/header.tsx

```diff
@@ -0,0 +1,97 @@
+import { GetHelpLink } from "@hashintel/design-system";
+import {
+  Box,
+  inputBaseClasses,
+  selectClasses,
+  Typography,
+} from "@mui/material";
+import { FunctionComponent, useState } from "react";
+
+import { ChatModelId, ChatModelSelector } from "./chat-model-selector";
+import { SystemPromptId, SystemPromptSelector } from "./system-prompt-selector";
+
+export const Header: FunctionComponent<{
+  disabled: boolean;
+  chatModel: ChatModelId;
+  setChatModel: (chatModel: ChatModelId) => void;
+  systemPromptId: SystemPromptId;
+  setSystemPromptId: (systemPromptId: SystemPromptId) => void;
+}> = ({
+  disabled,
+  chatModel,
+  setChatModel,
+  systemPromptId,
+  setSystemPromptId,
+}) => {
+  const [chatModelSelectorOpen, setChatModelSelectorOpen] = useState(false);
+  const [systemPromptSelectorOpen, setSystemPromptSelectorOpen] =
+    useState(false);
+
+  return (
+    <Box
```

Comment:
We probably want to make this header fade in/out as we hover in/out of the block, like other blocks. Right @nonparibus?

Also, I noticed the selectors are aligned to the right-side of the block, whereas in other blocks everything is aligned to the start. Do we want to keep consistency here as well?

Reply:
Yes and yes

Reply:
I wasn't aware we were fading these in an out in other components, happy to do this. Regarding the alignment, this was my interpretation from [the designs](https://www.figma.com/file/T10d2P2iNb1LdRC5a9J7vA/Blocks?node-id=1880-6619&t=yx3HPCB4uXm42dz3-0) but am realising now it's a bit ambiguous whether they should be right-aligned or left-aligned.